content/09.overfitting.md

```diff
@@ -1 +1,17 @@
 ## Tip 7: Address deep neural networks' increased tendency to overfit the dataset {#overfitting}
+
+Overfitting is one of the most significant dangers faced by a deep learning practitioner.
+Put simply, overfitting is when a model–deep or otherwise–memorizes the mapping from inputs to outputs rather than truly learning.
+This subtle distinction is made clearer by seeing what happens when a model is tested on data to which it was not exposed during training: just as a student who memorizes exam materials struggles to correctly answer questions for which they have not studied, a machine learning model that has overfit to its training data will perform poorly on unseen test data.
+Deep learning models are particularly susceptible to overfitting due to their greater representational capacity.
+To continue the student analogy, a smarter student has greater potential for memorization than average one and thus may be more inclined to memorize.
+
+The simplest way to combat overfitting is to detect it.
+This can be done by splitting the dataset into three parts: a training set, a tuning set (also commonly called a validation set), and a testing set.
```

Comment:
> "test set" sounds more natural

I disagree since its a parallel construction with "tuning" and "training" data

Reply:
sure, but I think here these don't act as gerunds but rather technical terms. Never seen "training and independent testing sets" -- usually it's "training and independent test set". But this is a minor thing and either way is fine since both ways are valid for different reasons :P

Reply:
Well, it seems [Wikipedia is on your side](https://en.wikipedia.org/wiki/Training,_validation,_and_test_sets).